### PR TITLE
Pull request for WAZO-1644-agent-login-choose-line

### PR DESCRIPTION
**Pull request by sduthil**
Depends-On: https://github.com/wazo-platform/wazo-agentd-client/pull/8
Depends-On: https://github.com/wazo-platform/xivo-dao/pull/62

### CODE REVIEW

File: wazo_agentd/http.py [hunk 1]
```diff
@@ -178,21 +156,33 @@ def get(self, agent_number):
...
+            agent_id, body['extension'], body['context'], tenant_uuids=tenant_uuids
         )
         return '', 204
 
 
 class _LoginAgentByNumber(_BaseResource):
     @required_acl('agentd.agents.by-number.{agent_number}.login.create')
     def post(self, agent_number):
-        extension, context = _extract_extension_and_context()
+        body = agent_login_schema.load(request.get_json(force=True))
```
Comment (fblackburn1) [inline wazo_agentd/http.py:170]: No need to add force=True
BaseSchema already handle this [here](https://github.com/wazo-platform/xivo-lib-python/blob/master/xivo/mallow_helpers.py#L82)

**Comment by sduthil** [comment]
recheck

File: wazo_agentd/http.py (hunk 1 above)
Comment (sduthil) [reply wazo_agentd/http.py]: It's fixed. Thank you!

**Comment by sduthil** [comment]
> Miss `integration_tests/suite/helpers/associations.py` file
> And the CHANGELOG is missing too

All fixed, thanks!

**Comment by sduthil** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Comment by fblackburn1** [comment]
recheck

**Pull request merged**